guide/src/documentation/commands/fee/register-counterparty-payee.md

```diff
@@ -0,0 +1,21 @@
+# Register Counterparty Payee
+
+Use this command in order to specify the address which will receive the `recv_fee` from incentivised packets relayed by the specified chain on the specified channel.
+
+__NOTE:__ Calling this command before starting the Hermes instance with the configuration `auto_register_counterparty_payee = true` set is useless as the `auto_register_counterparty_payee` will overwrite the address registered with the command.
```

Comment:
This sentence is a bit hard to parse. I think something like this would be easier to read:
```suggestion
__NOTE:__ Make sure the Hermes configuration parameter `auto_register_counterparty_payee = true` is set before calling this command, otherwise `auto_register_counterparty_payee` will overwrte the address registered with the command.
```